invenio_app_rdm/static_pages/static_pages.py

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2022 CERN.
+#
+# Invenio App RDM is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Static pages."""
+import os
+from pathlib import Path
+
+from flask_webpackext import current_webpack
+from invenio_db import db
+from invenio_pages import Page
+
+
+class StaticPages:
+    """Static pages."""
+
+    def page_data(self, page):
+        """Returns content of template."""
+        assets_path = current_webpack.project.path
```

Comment:
minor: I'm not sure if pages should go under assets/static. In this context, we're loading the pages similarly to fixtures, so [`app_data`](https://github.com/inveniosoftware/invenio-rdm-records/blob/fbc59a9ed17b6086930cadce077c07cb752f7481/invenio_rdm_records/fixtures/__init__.py#L40) might be the more "usual" place.